ui/package.json

```diff
@@ -70,7 +73,7 @@
     "ember-test-selectors": "^6.0.0",
     "ember-tooltips": "^3.5.1",
     "ember-truth-helpers": "^3.0.0",
-    "eslint": "^8.7.0",
+    "eslint": "^7.32.0",
```

Comment:
I'm wondering if the eslint downgrade is needed or if we could also stick to the `8+` version - what do you think?

Reply:
It was needed unfortunately. There is a mismatch with eslint and a babel dependency that ember relies on.

Reply:
Got it and thank you for outlining!